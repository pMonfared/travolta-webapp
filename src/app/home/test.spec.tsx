import * as React from 'react';
import renderer from 'react-test-renderer';

import Home from './page';

describe('Home', () => {
  test('snapshot renders', () => {
    const component = renderer.create(<Home />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
