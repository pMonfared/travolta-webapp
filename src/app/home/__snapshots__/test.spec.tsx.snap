// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home snapshot renders 1`] = `
<main
  className="flex min-h-screen flex-col items-center justify-between p-24"
>
  <h3>
    Hello World!
  </h3>
</main>
`;
